pkg/cmd/printer/printer.go

```diff
@@ -338,7 +338,7 @@ func (p tableEventPrinter) Print(event trace.Event) {
 			)
 		}
 	}
-	for i, arg := range event.Args {
+	for i, arg := range event.ParsedArgs {
```

Comment:
Is this always initialized for table printer?

Reply:
Yes.

![image](https://github.com/aquasecurity/tracee/assets/3372117/b874f923-9ee2-4637-abe6-70c9332b0e98)


To avoid this we need to set other output option like `-o json`.